app/_hub/kong-inc/vault-auth/index.md

```diff
@@ -33,39 +32,46 @@ params:
     - name: access_token_name
       required: true
       default: "`access_token`"
+      datatype: array of string elements
       description: |
-        Describes an array of comma-separated parameter names where the plugin will look for an access token. The client must send the access token in one of those key names, and the plugin will try to read the credential from a header or the querystring parameter with the same name.<br>*note*: the key names may only contain [a-z], [A-Z], [0-9], [_] and [-].
+        Describes an array of comma-separated parameter names where the plugin will look for an access token. The client must send the access token in one of those key names, and the plugin will try to read the credential from a header or the querystring parameter with the same name. The key names can only contain [a-z], [A-Z], [0-9], [_], and [-].
     - name: vault.id
       required: true
       default:
       value_in_examples: "<UUID>"
+      datatype: foreign ID
```

Comment:
What exactly is a "foreign ID" datatype? We should really try to standardize here. 

Looking at your conversation earlier, I see:

> I'm not aware yet of any other plugin that has this so we are creating a convention now. I think foreign ID is better. There is no mention of UUID in the description. But the existing example shows UUID. I'll go with whatever you think is best.

There is no UUID in the description, but the value in examples one line above is a UUID. The ID in question is a UUID, so can we specify that?